pkg/bmc/idrac_virtualmedia.go

```diff
@@ -90,3 +92,52 @@ func (a *redfishiDracVirtualMediaAccessDetails) VendorInterface() string {
 func (a *redfishiDracVirtualMediaAccessDetails) SupportsSecureBoot() bool {
 	return true
 }
+
+func (a *redfishiDracVirtualMediaAccessDetails) ValidateBootInterface(bmcCreds Credentials) (bool, error) {
```

Comment:
This logic is not Dell specific and must be replicated in redfish.

Reply:
It's specific for virtual media with redfish, so doesn't make sense to change `pkg/bmc/redfish.go` because it's `ipxe`.

Reply:
Yes, I meant redfish-virtualmedia

Reply:
It sounds like the logic could be moved to a private function in this package and called from both drivers.

Reply:
Doug, something like `virtualmedia.go` in `pkg/bmc/utils`?

Reply:
Maybe `pkg/bmc/virtualmedia_validation.go`? That would mean the function could be private to the package that uses it.

Reply:
Awesome, Thanks!

Reply:
Done